hwtracer/src/decode/ykpt/mod.rs

```diff
@@ -415,16 +413,16 @@ impl<'t> YkPTBlockIterator<'t> {
                     // If there are calls in the block that come *after* the current position in the
                     // block, then we will need to follow those before we look at the successor info.
                     if let Some(blk) = self.maybe_follow_blockmap_call(b_off, &ent.value)? {
-                        return Ok(blk);
+                        Ok(blk)
```

Comment:
I don't prefer this, but if that's what clippy suggests.

Reply:
I have a marginal preference for the adjusted version, just because it makes clear that this isn't early termination. [And, as you say, Clippy does suggest it!]